libs/utils/analysis_module.py

```diff
@@ -41,4 +44,7 @@ def __init__(self, trace):
 
         trace._registerDataFrameGetters(self)
 
+        logging.config.fileConfig('logging.conf')
```

Comment:
As discussed by voice, since there can be dependencies among different modules, maybe it's not simple to find a single point where the logfile can be configured once forever.
I would propose to keep it simple at this first attempt and then eventually refactor it in a second stage.